bayesian/v2/__init__.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
can the package be api instead of v2, the file name is already saying v2

Reply:
I'm concerned about this. 
Keeping common Utils file for each version(v2, v3...) will eventually lead to Low Maintainability Index of each file. We already have low MI for `utils.py` but keeping it common will lead to each version dumping its new Utility functions to utility files, which further Degrade Numbers in MI. 
This is the main reason for namespacing in folders v2, v3 like so and enabling separate utility files.